dmutils/email/user_account_email.py

```diff
@@ -0,0 +1,40 @@
+from flask import current_app, session, abort
+from . import DMNotifyClient, generate_token, EmailError
+from .helpers import hash_string
+
+
+class UserAccountEmail():
```

Comment:
This feels like this could be a function. Class attributes are essentially used as arguments for the single method.
Do you think this will get extended in the future in a way that would be easier to do with a class?

Reply:
Have updated to not be a class, and have imported the external blueprint from the supplier frontend.